standard/lua.lua

```diff
@@ -194,4 +197,48 @@ function Lua.getDefaultEntryPoints(module)
 	return fnNames
 end
 
+--[==[
+Wrapper for mass LPDB queries.
```

Comment:
Doc doesn't actually answer the "why" of this function

Reply:
updated it a bit, hope that explains the why good enough

Reply:
Aside from the fact that it's now gone, no it did not help me. Looks to me like you're making a bunch of queries in a loop, so I'm confused.

Reply:
yeah reworked entirely, hence removed the comments until finished
readded some now